ci/env.sh

```diff
@@ -31,9 +31,20 @@ if [ -z $DOCKERHUB_ORG ]; then
     export DOCKERHUB_ORG=appsody
 fi
 
+# find github repository slug
+if [ -z "$TRAVIS_REPO_SLUG" ]
+then
+  git_org=$(git branch -vv | grep -e '^\*.*' | cut -d' ' -f4 | cut -d'[' -f2 | cut -d'/' -f1)
+  export REPO_SLUG=$(git remote get-url $git_org | sed -e 's#git@\(.*\):\([^.]*\)\(\.git\)\?#\2#' -e 's#https://\([^/]*\)/\([^.]*\)\(\.git\)\?#\2#')
```

Comment:
Do we really need to do all this grep / cut? or can we expect users to set a variable or parr an argument to the build scripts?

Reply:
This is pulling information about the local repo, _either_ from a build variable (as TRAVIS passes) OR from the remote associated with the current git branch (which might not be origin)